pom.xml

```diff
@@ -185,6 +185,12 @@
             <version>1.61</version>
         </dependency>
 
+        <dependency>
+            <groupId>postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>9.1-901-1.jdbc4</version>
```

Comment:
This is an old version. You should use https://mvnrepository.com/artifact/org.postgresql/postgresql/42.2.18 instead. Unfortunately, this change requires you to validate everything again